www/~/%username/emails/index.spt

```diff
@@ -1,12 +1,18 @@
-from gratipay.utils import get_participant
+from gratipay.utils import get_participant, icons
 
 [-----------------------------------------------------------------------------]
 
 participant = get_participant(state, restrict=True)
 banner = '~' + participant.username
-title = _("Emails")
+title = _("Email")
 
 emails = participant.get_emails()
+groups = [[], [], []]  # primary, verified, unverified
+for email in emails:
+    i = 0 if email.address == participant.email_address else 1 if email.verified else 2
```

Comment:
We're essentially doing a `sort_by(func)` action here, can't we use https://docs.python.org/3/library/functions.html#sorted with the `key` param? 

i.e. something like: 
```python
def get_sort_key(email):
  if email.address == participant.email_address:
    return 0
  elif email.verified:
     return 1
  else:
    return 2

sorted_emails = sorted(emails, key=get_sort_key)
```

Reply:
Looks less cryptic to me than the awkward spelling of `flattern`, and the inlined inline `if`

Reply:
Done in https://github.com/gratipay/gratipay.com/pull/4579/commits/11a93a4b3943a1ed5574d6cb1e11f46d20e65c4c